04/app.js

```diff
@@ -13,13 +13,13 @@ function init() {
   });
 
   rangeInput.addEventListener("change", function (e) {
-    // let setOpacity = e.target.value;
-    boxElement.style.opacity = e.target.value / 100;
+    let boxOpacity = (boxElement.style.opacity = e.target.value / 100);
+    setBoxShadow(boxElement, boxOpacity);
```

Comment:
Zwróć uwagę, że `setBoxShadow` otrzymuje 3 parametry - 1 to element, 2 to kolor, 3 to dopiero opacity.
Dlatego prawidłowe użycie to: `setBoxShadow(boxElement, colorInput.value, boxOpacity);`

Reply:
👍